reference/api/settings.md

```diff
@@ -1,8 +1,8 @@
 # All settings
 
-`Settings` is a list of all the **customization** possible for an index.
+Index settings are represented as a [JSON object literal](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON), containing a field for each possible customization option.
```

Comment:
Given how index can be a lot of things in programming (e.g. an array index), it could be useful to make this explicit. What do you think about linking to `/learn/core_concepts/indexes.md`?

```suggestion
[Index](/learn/core_concepts/indexes.md) settings are represented as a [JSON object literal](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON), containing a field for each possible customization option.
```

Reply:
Sure, let's go with this. The other option would be adding a glossary tooltip, but that's not great for accessibility reasons.